init_scripts/init-root.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+echo "nameserver 8.8.8.8" | sudo tee /etc/resolv.conf
```

Comment:
Can you elaborate why are we writing both to standard output and the file `resolve.conf`? What is the purpose of this file? maybe it's worth adding a comment

Reply:
8.8.8.8 is Google's DNS server, this line basically resets the dns server to avoid issues with virtualboxe's default nat assigned dns server (which sometimes causes issues, as it did on my machine)

Reply:
You really shouldn't mess with that - its one of those things that may break on other people's machines.